### PR TITLE
Vue: Fix get version template

### PR DESCRIPTION
**Pull request by OlegKipchatov**

### CODE REVIEW

File: packages/devextreme-cli/src/templates/vue-v2/application/devextreme.json [hunk 1]
```diff
@@ -1,5 +1,6 @@
 {
   "applicationEngine": "vue",
+  "version": "2",
```
Comment (babich-a) [inline packages/devextreme-cli/src/templates/vue-v2/application/devextreme.json:3]: I do not like this format. 'version' is the first-level option now but affected only vue template now. If we propose a template-based option here, we should separate them. 
e.g. `{ "applicationEngine": "vue", "vue": { "version": 2 }}`

Comment (OlegKipchatov) [reply packages/devextreme-cli/src/templates/vue-v2/application/devextreme.json:3]: maybe `"option": { "version": 2 }`?

Comment (babich-a) [reply packages/devextreme-cli/src/templates/vue-v2/application/devextreme.json:3]: `option` is the common. My idea `devextreme[devextreme["applicationEngine"]].version`. 'version' in this case is specific for `devextreme["applicationEngine"]`

File: packages/devextreme-cli/src/applications/application.vue.js [hunk 2]
```diff
@@ -17,16 +17,11 @@ const defaultStyles = [
...
-        const keyValue = Object.keys(dependencies).filter(k => k.startsWith('vue@')).pop();
-        const vueVersion = dependencies[keyValue].version;
-        const majorVueVersion = vueVersion.split('.')[0];
-        return `v${majorVueVersion}`;
-    }
-
-    return defaultVueVersion;
+    const devextremeConfig = require('../utility/devextreme-config').read();
+    
+    return devextremeConfig.version
```
Comment (babich-a) [inline packages/devextreme-cli/src/applications/application.vue.js:22]:
```suggestion
    return devextremeConfig.vue
```

Comment (babich-a) [inline packages/devextreme-cli/src/applications/application.vue.js:22]:
```suggestion
    return devextremeConfig.vue.version
```
smth like this?

**Review by babich-a — APPROVED** [review]

**Review by egaluza — APPROVED** [review]

**Pull request merged**